utils/libnetwork/Makefile

```diff
@@ -11,11 +10,11 @@ GO_PKG_BUILD_PKG:= \
   $(GO_PKG)/cmd/proxy \
   $(GO_PKG)/cmd/dnet
 
+PKG_SOURCE_VERSION:=fc5a7d91d54cc98f64fc28f9e288b46a0bee756c
```

Comment:
erm, put these after PKG_SOURCE_URL. PKG_SOURCE_DATE before PKG_SOURCE_VERSION

Reply:
In order not to have this be changing on the whim of person to person I've elected to follow the order of definition in the closest to an officially documented style guide I could find at the time of writing, https://openwrt.org/docs/guide-developer/packages#BuildPackage_variables.

Reply:
https://github.com/openwrt/packages/issues/9682

Reply:
I think the pull request is now in fairly good shape to be merged.

The look and feel thing is no blocker.  Even if we later agreed on a such style guide about order, that should be managed by a tool and convert the tree as a treewide.